src/pages/MovieCart/MovieCart.js

```diff
@@ -4,100 +4,100 @@ import MovieItem from './MovieItem';
 import MovieReceipt from './MovieReceipt';
 
 const MovieCart = () => {
-  const [movieList, setMovieList] = useState([]);
-  const accessToken = localStorage.getItem('token') ?? '';
+  const [pickMovieList, setPickMovieList] = useState([]);
+  const [orderMovieList, setOrderMovieList] = useState([]);
 
-  const totalSumPrice = movieList.reduce(
-    (sum, current) => sum + current.price * current.quantity,
+  const accessToken = localStorage.getItem('token') ?? '';
+  const totalSumPrice = pickMovieList.reduce(
+    (sum, current) => sum + current.price * current.counts,
     0
   );
 
   const itemPlusCount = (id) => {
-    const next = movieList.map((movieItem) => {
+    const next = pickMovieList.map((movieItem) => {
       if (movieItem.id === id) {
-        return { ...movieItem, quantity: movieItem.quantity + 1 };
+        return { ...movieItem, counts: movieItem.counts + 1 };
       }
       return movieItem;
     });
-    setMovieList(next);
+    setPickMovieList(next);
   };
 
   const itemMinusCount = (id) => {
-    const next = movieList.map((movieItem) => {
+    const next = pickMovieList.map((movieItem) => {
       if (movieItem.id === id) {
-        if (movieItem.quantity === 1) {
-          return { ...movieItem, quantity: 1 };
+        if (movieItem.counts === 1) {
+          return { ...movieItem, counts: 1 };
         } else {
-          return { ...movieItem, quantity: movieItem.quantity - 1 };
+          return { ...movieItem, counts: movieItem.counts - 1 };
         }
       }
       return movieItem;
     });
-    setMovieList(next);
+    setPickMovieList(next);
   };
 
-  const getMovieData = () => {
-    fetch(`/data/movie.json`, {
+  const getPickMovieData = () => {
+    fetch(`http://10.58.52.61:3000/myShallWe/pick`, {
       method: 'GET',
-      headers: { authorization: accessToken },
+      headers: {
+        authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsImlhdCI6MTY3NTU4NDcyNX0.tylNeMdSFT6iAxe6kJyfuoDpKduEoik4kTYM_PfiW1c',
+      },
     })
       .then((response) => response.json())
-      .then((res) => setMovieList(res));
+      .then((res) => setPickMovieList(res.data));
   };
   useEffect(() => {
-    getMovieData();
+    getPickMovieData();
   }, []);
 
   const deleteItem = (id) => {
-    fetch(`/movies/${id}`, {
+    const resultArray = [...pickMovieList];
+    pickMovieList.forEach(function (item, index) {
+      if (item.id === id) {
+        resultArray.splice(index, 1);
+      }
+    });
+    setPickMovieList([...resultArray]);
```

Comment:
```suggestion
    setPickMovieList(pickMovieList.filter(item => item.id === id));
```
filter를 통해서 작업하면 줄이 많이 줄어들 수 있지 않을까요?

Reply:
오오 감사합니다!!! 수정해서 반영했습니다!